bciers/apps/coam/app/page.tsx

```diff
@@ -0,0 +1,467 @@
+export default function Index() {
+  /*
+   * Replace the elements below with your own.
+   *
+   * Note: The corresponding styles are in the ./index.css file.
+   */
+  return (
+    <div>
+      <div className="wrapper">
```

Comment:
Maybe remove all this Nx boilerplate HTML and just put something generic like:

`<h1>COAM</h1>` or whatever placeholder you think is best!